tests/phpunit/elementor/modules/landing-pages/test-module.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace Elementor\Testing\Modules\LandingPages;
+
+use Elementor\Testing\Elementor_Test_Base;
+use Elementor\Modules\LandingPages\Module;
+
+class Elementor_Test_Landing_Pages_Module extends Elementor_Test_Base {
+	/** @var Module */
+	private static $module;
+
+	public function test_should_confirm_module_activation() {
```

Comment:
Change this method name and don't initialize a new module

Reply:
Done.